src/tools/rest-backend/.gitignore

```diff
@@ -0,0 +1,3 @@
+CMakeCache.txt
+nbproject/
```

Comment:
> nbproject

I meant to simply add a comment next to the line.

> No, I did not use in-source builds for the backend. Why?

This was suggested by ignoring CMakeCache.txt.